Octokit/Clients/RepositoryInvitationsClient.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Octokit
+{
+    public class RepositoryInvitationsClient : ApiClient, IRepositoryInvitationsClient
+    {
+        public RepositoryInvitationsClient(IApiConnection apiConnection)
+            : base(apiConnection)
+        {
+        }
+
+        /// <summary>
+        /// Accept a repository invitation.
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://developer.github.com/v3/repos/invitations/#accept-a-repository-invitation">API documentation</a> for more information.
+        /// </remarks>        
+        /// <param name="invitationId">The id of the invitation</param>
+        /// <exception cref="ApiException">Thrown when a general API error occurs.</exception>        
+        public async Task<bool> Accept(int invitationId)
+        {
+            var endpoint = ApiUrls.UserInvitations(invitationId);
+
+            try
+            {
+                var httpStatusCode = await ApiConnection.Connection.Patch(endpoint, AcceptHeaders.InvitationsApiPreview).ConfigureAwait(false);
+                return httpStatusCode == HttpStatusCode.NoContent;
+            }
+            catch
```

Comment:
For consistency with other simliar clients/endpoints, this should be 
``` csharp
catch (NotFoundException)
```

rather than catching any type of exception